src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -1662,6 +1663,39 @@ GUI_TEST_CLASS_DEFINITION(test_7451) {
     GTUtilsStartPage::checkRecentListUrl(os, "test_7451.fa", false);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7454) {
+    // Open data/samples/PDB/1CF7.PDB.
+    // Increase the width of the Project View.
+    // Open any tab in the Options Panel and increase its width.
+    //     Expected: the current Sequence View is narrow.
+    // Find the action toolbar extension for the first sequence in the Sequence View (">>" button). Press ">>"->
+    //         "X Remove sequence".
+    //     Expected: no crash.
+    GTUtilsProject::openFile(os, dataDir + "samples/PDB/1CF7.PDB");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    QPoint splitterCenter = [&os] {
+        auto projectView = GTWidget::findWidget(os, "project_view");
+        QRect rect = projectView->geometry();
+        return projectView->mapToGlobal({rect.right() + 4, rect.center().y()});
+    }();
+    auto delta = [&os]() -> QPoint {
```

Comment:
replace the following 5 lines with a new method ```GTMainWindow::getMainWindowWidgetByName(os, name)``` 1-liner.
This method may be re-usable in other places.

Reply:
✓